external/future_update

```diff
@@ -0,0 +1,15 @@
+function get_versions(){
```

Comment:
Please add a comment to describe the purpose of this function.  Since currently it is not called, there is no way for us to test it in automation.  I believe @sophia-guo was suggesting this function get moved to common_functions.sh and the supported versions line get updated to call it.  

Once this is done, we would have to test it on all platforms where these tests are expected to run.  I can give instructions on how you will do that once this PR is updated.

Reply:
I hope it fulfills your request.